git-branchless-record/tests/test_record.rs

```diff
@@ -279,6 +279,66 @@ fn test_record_detach() -> eyre::Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_record_stash() -> eyre::Result<()> {
+    let git = make_git()?;
+
+    if !git.supports_reference_transactions()? {
+        return Ok(());
+    }
+    git.init_repo_with_options(&GitInitOptions {
+        make_initial_commit: false,
+        run_branchless_init: false,
+    })?;
+    git.branchless("init", &["--main-branch", "master"])?;
```

Comment:
It looks like the test is failing now. You probably want to just initialize the repository normally? Currently, if no initial commit is made, then the `master` branch can't be found.

Reply:
It worked on my machine, I don't exactly know why, but now I made it to include the root commit.